src/app/app.module.ts

```diff
@@ -74,6 +74,10 @@ import { SideBarComponent } from './components/utility/side-bar/side-bar.compone
 import { MatMenuModule } from '@angular/material/menu';
 import { MatIconModule } from '@angular/material/icon';
 
+import {PasswordMismatchValidatorDirective} from './Directives/password.validator';
```

Comment:
```suggestion
import { PasswordMismatchValidatorDirective } from './Directives/password.validator';
```
Have consistent spacing for imports. (Also at other places.)

Reply:
Resolve this?

Reply:
done